sparkle/src/components/BuilderLayout.tsx

```diff
@@ -8,48 +8,56 @@ import { Button } from "./Button";
 interface BuilderLayoutProps {
   leftPanel: React.ReactNode;
   rightPanel: React.ReactNode;
+  isRightPanelOpen: boolean;
+  toggleRightPanel: () => void;
 }
 
-export function BuilderLayout({ leftPanel, rightPanel }: BuilderLayoutProps) {
-  const [isRightPanelOpen, setIsRightPanelOpen] = useState(true);
-
-  const toggleRightPanel = () => {
-    setIsRightPanelOpen(!isRightPanelOpen);
-  };
-
+export function BuilderLayout({
```

Comment:
- made it non-managed
- removed the background colors
- made it work on mobile (no right panel / preview + no button)
- tweaked the sizes

Reply:
:+1: 
But in the stories.tsx can you put background colors & lorem ipsum content so the sparkle doc shows better if the preview hides or pushes content, and/or how it's transitioned if it is?